### ISSUE TITLE
Readme: the URL for Groemping (2021) holds an unintended backslash

### ISSUE THREAD

**Issue by ugroempi** [bertcarnell/SOAs#19]
Hi Rob,
in the manual, the link works OK, but in the Readme, an underscore in the URL is escaped by a backslash (both display and link), which breaks the link. Do you know how to fix that?
Best, Ulrike

**Comment by bertcarnell** [comment]
Here is what I see:

`SOAs-package.R` and `SOAs-package.Rd` have 

```
\url{http://www1.beuth-hochschule.de/FB_II/reports/Report-2021-001.pdf}
```

Gets translated to the `README.md` with the underscore escaped in the display html, but not the `href`

```
<a href="http://www1.beuth-hochschule.de/FB_II/reports/Report-2021-001.pdf">http://www1.beuth-hochschule.de/FB\_II/reports/Report-2021-001.pdf</a>
```

When I run it interactively in RStudio, I don't get the escaped underscore.  (R4.0.4 and RStudio 1.4.1106).  When I run `rmarkdown::render("README.Rmd")`, I do get the escaped underscore.

This [link](https://stackoverflow.com/questions/35827838/how-to-show-underscores-symbol-in-markdown) is relevant.

I think it happens in the pandoc translation to markdown.

I'll try replacing it with a html character for underscore.

**Comment by bertcarnell** [comment]
I implemented a partial fix with c501ec88edf5df450ae56c0a3d427fce0b9c6a25

I enclosed the url in backticks so that pandoc would treat the text as a literal.  It does put "color" the text a little since is then a code block.  Let me know what you think.

**Comment by ugroempi** [comment]
I think it's a good pragmatic fix for the specific url. It looks a bit odd, but that shouldn't be a problem.
